### PR TITLE
Fix pulling licenses

### PR DESCRIPTION
**Pull request by damccorm**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by damccorm** [comment]
Run Python_PVR_Flink PreCommit

File: sdks/java/container/license_scripts/dep_urls_java.yaml [hunk 1]
```diff
@@ -65,3 +65,7 @@ org.eclipse.jgit:
   '4.4.1.201607150455-r':
     license: "https://www.eclipse.org/org/documents/edl-v10.html"
     type: "Eclipse Distribution License - v1.0"
+zstd-jni:
+  '1.5.2-5':
+    license: "https://github.com/luben/zstd-jni/blob/master/LICENSE"
```
Comment (bvolpato) [inline sdks/java/container/license_scripts/dep_urls_java.yaml:70]: Should this be `https://raw.githubusercontent.com/luben/zstd-jni/master/LICENSE`?
```suggestion
    license: "https://raw.githubusercontent.com/luben/zstd-jni/master/LICENSE"
```

Comment (damccorm) [reply sdks/java/container/license_scripts/dep_urls_java.yaml]: Yes, updated

**Comment by johnjcasey** [comment]
LGTM

**Comment by johnjcasey** [comment]
Run Java Examples on Dataflow Runner V2

**Comment by johnjcasey** [comment]
run java precommit

**Comment by pabloem** [comment]
Run Java PreCommit

**Review by bvolpato — APPROVED** [review]

**Comment by damccorm** [comment]
Self hosted runners are down which is blocking the checks here, I'm not going to block on that since this clearly fixes the problem

**Pull request merged**